modules/api/api-xsjs/src/main/java/com/sap/xsk/api/destination/CloudPlatformDestinationFacade.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.api.destination;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.sap.cloud.sdk.cloudplatform.CloudPlatformAccessor;
+import com.sap.cloud.sdk.cloudplatform.connectivity.Destination;
+import com.sap.cloud.sdk.cloudplatform.connectivity.DestinationAccessor;
+import com.sap.cloud.sdk.cloudplatform.connectivity.HttpClientAccessor;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.MethodNotSupportedException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpHead;
+import org.apache.http.client.methods.HttpOptions;
+import org.apache.http.client.methods.HttpPatch;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.client.methods.HttpRequestBase;
+import org.apache.http.client.methods.HttpTrace;
+import org.apache.http.util.EntityUtils;
+import org.eclipse.dirigible.commons.api.scripting.IScriptingFacade;
+import java.io.IOException;
+import java.net.URI;
+import java.util.Arrays;
+
+public class CloudPlatformDestinationFacade implements IScriptingFacade {
+
+  private static String QUERY_PATH = "queryPath";
+  private static String METHOD = "method";
+  private static String HEADERS = "headers";
+  private static String STATUS_CODE = "statusCode";
+  private static String TEXT = "text";
+  private static String NAME = "name";
+  private static String VALUE = "value";
+
+  public static Destination getDestination(String name) {
+    setKymaCloudPlatformFacade();
+
+    return DestinationAccessor.getDestination(name);
+  }
+
+  public static String executeRequest(String requestObject, Destination destination) throws IOException, MethodNotSupportedException {
+    setKymaCloudPlatformFacade();
+
+    JsonParser jsonParser = new JsonParser();
+    JsonObject requestJson = jsonParser.parse(requestObject).getAsJsonObject();
```

Comment:
Could use GSON to serialize/deserialize JSON objects, that POJO objects would be:

```java
requestJson.get(QUERY_PATH).getAsString()
```

->

```java
request.getQueryPath()
``